OrbitCore/Path.h

```diff
@@ -9,8 +9,6 @@
 #include <vector>
 
 namespace Path {
```

Comment:
Do you plan to remove/move this in the future, or do we want to keep OrbitCore::Path for now?
In the latter case, consider renaming the OrbitCore version.

Reply:
I am thinking about moving it too.. this is for follow up cl